packages/core/src/db-utils.ts

```diff
@@ -0,0 +1,55 @@
+import {PromiseReturnType} from "types"
+import {PaginationPayloadInvalidError} from "./errors"
+
+type SeederOptions<T> = {
+  amount?: number
+  options?: T
+}
+
+type Seeder<T extends Promise<any>, U> = (index: number, options?: U) => T
+
+export const seeder = <T extends Promise<any>, U>(seeder: Seeder<T, U>) => ({
+  seed: async (options?: SeederOptions<U>) => {
+    const seeds: PromiseReturnType<typeof seeder>[] = []
+
+    for (const index in [...Array(options?.amount || 1)]) {
+      const data = await seeder(Number(index), options?.options)
+
+      seeds.push(data)
+    }
+
+    return seeds
+  },
+})
+
+export const paginate = async <T extends Promise<object[]>>({
+  skip,
+  take,
+  takeMax = 100,
```

Comment:
What do you think about increasing this default to maybe 500? 100 seems lower than necessary and could be annoying when people try to use more than 100 in their code.